### PR TITLE
Jormungandr: rename experimental to distributed

### PR DESCRIPTION
**Pull request by xlqian**
rename experimental to distributed

### CODE REVIEW

File: source/jormungandr/jormungandr/instance.py [hunk 1]
```diff
@@ -151,6 +151,8 @@ def scenario(self, override_scenario=None):
             logger = logging.getLogger(__name__)
             logger.debug('overriding the scenario for %s with %s', self.name, override_scenario)
             try:
+                # for the sake of backwards compatibility... some users may still be using experimental...
+                override_scenario = 'distributed' if override_scenario == 'experimental' else override_scenario
                 module = import_module('jormungandr.scenarios.{}'.format(override_scenario))
             except ImportError:
                 logger.exception('sceneario not found')
```
Comment (pbougue) [inline source/jormungandr/jormungandr/instance.py]: `scenario` (no `e`)

**Review by pbougue — APPROVED** [review]

**Review by pbougue — CHANGES_REQUESTED** [review]
Not done before, but it would be nice to add it to tyr: https://github.com/CanalTP/navitia/blob/dev/source/tyr/tyr/resources.py#L386

**Review by pbougue — APPROVED** [review]

**Comment by pbougue** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**